### PR TITLE
re-implement v2 branch as 2.x

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
-  });
-});
+const iteratorStream = require('level-iterator-stream')
+const leveldown = require('leveldown')
+
+const db = leveldown(__dirname + '/db')
+db.open(function (err) {
+  if (err) throw err
+
+  var stream = iteratorStream(db.iterator())
```
Comment (ralphtheninja) [inline README.md]: Gah, missed the `const` again :)

**Comment by ralphtheninja** [comment]
@vweevers Pulling you in as well as a reviewer, so you know what's going on.

**Comment by ralphtheninja** [comment]
Once new major is released, I'll pull that version into `2.x` branch of `levelup`.

File: README.md [hunk 2]
```diff
@@ -3,25 +3,24 @@
...
-  if (err) throw err;
-
-  var stream = iteratorStream(db.iterator());
-  stream.on('data', function(kv){
-    console.log('%s -> %s', kv.key, kv.value);
-  });
-});
+const iteratorStream = require('level-iterator-stream')
+const leveldown = require('leveldown')
+const db = leveldown(__dirname + '/db')
```
Comment (juliangruber) [inline README.md]: can we keep the blank line between require statements and logic?

Comment (ralphtheninja) [reply README.md]: Of course!

**Pull request merged**